requirements.txt

```
@@ -20,7 +20,7 @@ python-dotenv
 djangorestframework
 git+https://github.com/noaa-owp/hypy@master#egg=hypy&subdirectory=python
 hydrotools.nwis-client
-numpy
+numpy==1.26.4
```

I think adding this alone should resolve our issue. If a dependency requires `numpy>1`, things will start failing, but I don't think the arguments added above will address that either. I think it makes the most sense to add this check into `ngen` for the time being until support is added. That way, `ngen` just fails if it detects that `numpy>1` is being used. 

```suggestion
numpy~=1.0
```

---

I'm strongly leaning toward taking this out also.  Again, the real need was for ngen compilation inside the workers.  This goes beyond that.  Further, this file actually applies to the place where the user actually has (and likely is capable of handling) the most control over what numpy is installed.  I don't think we want to take that control away.

---

Yeah, sorry I wasn't paying the closest attention. I would advocate for creating a ngen worker `requirements.txt` (if needed) that we copy into the worker `dockerfile` and install if python is enabled. I think that is the easiest solution. Thoughts?